pkg/api/component_report.go

```diff
@@ -1647,6 +1671,13 @@ func (c *componentReportGenerator) generateComponentTestReport(baseStatus map[st
 		updateCellStatus(rowIdentifications, columnIdentification, testID, testStats, aggregatedStatus, allRows, allColumns, nil, c.openRegressions)
 	}
 
+	// Fallback for any open regressions in the db, that did not appear in this report.
+	// This would likely imply they no longer have sufficient sample, and this should be rare.
+	err := c.closeRegressionsForMissingTests(rLog, matchedOpenRegressions, now)
```

Comment:
Wondering how the Base sample plays into regression tracking.  If I change the base sample window to something small or less stable time period would that cause regressions to close for the main GA base sample time?  Should we consider differentiating open regressions based on the base sample release, start and end dates?

Reply:
Regression tracking should only be turned on in a very specific case which is the metrics loop, there should be no other way to turn it on. We should be able to assume that the base will remain fixed, and once server side views land that will be encoded in the view and should never change.